### PR TITLE
update: add standata to common config

### PR DESCRIPTION
**Pull request by VsevolodX**

### CODE REVIEW

File: config.yml [hunk 1]
```diff
@@ -9,6 +9,7 @@ default:
     - pandas==1.5.3
     - ase==3.22.1
     - ipywidgets
+    - mat3ra-standata
```
Comment (timurbazhirov) [inline config.yml:12]: Let's add to where it's required only

Comment (VsevolodX) [reply config.yml:12]: But if it's installed everywhere, it's faster to import and use in any notebook. you don't need to specify this, just import standata and use

File: config.yml [hunk 2]
```diff
@@ -64,3 +64,6 @@ notebooks:
     packages_pyodide:
       - mat3ra-esse
       - mat3ra-made
+  - name: specific_examples
```
Comment (timurbazhirov) [inline config.yml:67]: Let’s put a comment that this will be applied for the whole folder

**Review by timurbazhirov — APPROVED** [review]

File: config.yml (hunk 2 above)
Comment (VsevolodX) [reply config.yml:67]: no, you specify this line in the installation function

**Pull request merged**